org.jacoco.core.test/src/org/jacoco/core/analysis/AnalyzerTest.java

```diff
@@ -89,23 +89,6 @@ public void should_ignore_synthetic_classes() throws Exception {
 		assertTrue(classes.isEmpty());
 	}
 
-	@Test
-	public void should_analyze_java10_class() throws Exception {
-		final byte[] bytes = createClass(BytecodeVersion.V10);
-		final long expectedClassId = CRC64.classId(bytes);
-
-		analyzer.analyzeClass(bytes, "");
-
-		assertEquals(expectedClassId, classes.get("Foo").getId());
```

Comment:
This removes test that was checking that we use original version for `classId`

https://github.com/jacoco/jacoco/blob/68ab19f3463bf5076485185fe2abe5ec8a726679/org.jacoco.core/src/org/jacoco/core/analysis/Analyzer.java#L115